docs/docs/adding-page-transitions-with-plugin-transition-link.md

```diff
@@ -139,26 +140,26 @@ const Box = posed.div({
 </TransitionState>
 ```
 
-Now, the `Box` component will be aware of the transition status of the page it's a child of, and it will fade in/out accordingly.
+Now, the `Box` component will be aware of wether the page it's a child of is mounting or unmounting, and it will fade in/out accordingly.
```

Comment:
```suggestion
Now, the `Box` component will be aware of whether the page is a child of is mounting or unmounting, and it will fade in/out accordingly.
```